README.md

```diff
@@ -108,7 +112,12 @@ pip install dali-rp2
 ### Installation on Other Unix-like Systems
 * install python 3.8 or greater
 * install pip3
-* `pip install dali-rp2`
+
+Then install DaLI:
```

Comment:
```suggestion
Then install DaLI-RP2:
```

Reply:
There are other areas of README.md that say "Then install DaLI".  Would you like me to change them all?

Reply:
DaLI is the name of the software project, whereas dali-rp2 is the name of the Python package. I think in this context saying "Then install DaLI" is OK and shouldn't be too confusing. It's an English sentence directing the user to install the software, followed by the actual command to use in the CLI: I think the English sentence should use "DaLI" (which is less formal) and the command should (of course) use dali-rp2. Just my 2c.

Reply:
Keep as is then. Thanks for the fixes!